test/tests/ver-1jb/comparison_ver-1jb.py

```diff
@@ -0,0 +1,219 @@
+import matplotlib.pyplot as plt
+import numpy as np
+from matplotlib import gridspec
+import pandas as pd
+import os
+
+# Changes working directory to script directory (for consistent MooseDocs usage)
+script_folder = os.path.dirname(__file__)
+os.chdir(script_folder)
+
+# ===============================================================================
+# First Case - low mobile concentration
+# ===============================================================================
+
+# ===============================================================================
+# Extract TMAP8 predictions - time evolution
+if "/TMAP8/doc/" in script_folder:     # if in documentation folder
+    csv_folder = "../../../../test/tests/ver-1jb/gold/ver-1jb_time_dependent_out.csv"
+else:                                  # if in test folder
+    csv_folder = "./gold/ver-1jb_time_dependent_out.csv"
+tmap8_solution = pd.read_csv(csv_folder)
+tmap8_time = tmap8_solution['time']
+tmap8_tritium_mobile = tmap8_solution['tritium_mobile_inventory']
+tmap8_tritium_trapped = tmap8_solution['tritium_trapped_inventory']
+tmap8_helium = tmap8_solution['helium_inventory']
+
+# ===============================================================================
+# Case set up
+slab_length = 1.5 # m
+slab_height = 1 # m - assumed for integrations
+slab_width = 1 # m - assumed for integrations
+trapping_sites_atomic_fraction_max = 0.001 # at.frac.
+trapping_sites_fraction_occupied_initial = 0.5 # (-)
+density_material = 6.34e28 # atoms/m^3 # for tungsten
+normal_center_position =  slab_length/2 # m
+normal_standard_deviation = slab_length/4 # m
+
+# ===============================================================================
+# Calculate the analytical solution
+tritium_mobile_concentration_initial = 1 # atoms/m^3
+tritium_trapped_inventory_initial = 2.8438315780556e25 * slab_height * slab_width # atoms -- integral of the normal distribution over the slab length * area
+tritium_mobile_inventory_initial = tritium_mobile_concentration_initial * slab_length * slab_height * slab_width # atoms
+half_life = 12.3232 # years
+conversion_years_to_s = 365.25*24*60*60
+half_life_s = half_life*conversion_years_to_s # s
+decay_rate_constant = 0.693/half_life_s # 1/s
+
+analytical_tritium = (tritium_mobile_inventory_initial + tritium_trapped_inventory_initial) * np.exp(- decay_rate_constant * tmap8_time)
+analytical_helium = (tritium_mobile_inventory_initial + tritium_trapped_inventory_initial) * ( 1. - np.exp(- decay_rate_constant * tmap8_time))
+
+#  ===============================================================================
+# Plot figure for verification of tritium decay - time evolution
+fig = plt.figure(figsize=[6.5, 5.5])
+gs = gridspec.GridSpec(1, 1)
+ax = fig.add_subplot(gs[0])
+tmap8_time_years = [t/conversion_years_to_s for t in tmap8_time]
```

Comment:
```suggestion
tmap8_time_years = tmap8_time/conversion_years_to_s
```
If this breaks, tmap8_time.to_numpy()/conversion_years_to_s might be required instead.